src/Core/System.php

```diff
@@ -223,15 +223,30 @@ public static function currentLoad()
 	 * Redirects to an external URL (fully qualified URL)
 	 * If you want to route relative to the current Friendica base, use App->internalRedirect()
 	 *
-	 * @param string $url The new Location to redirect
+	 * @param string $url  The new Location to redirect
+	 * @param int    $code The redirection code, which is used (Default is 302)
+	 *
 	 * @throws InternalServerErrorException If the URL is not fully qualified
 	 */
-	public static function externalRedirect($url)
+	public static function externalRedirect($url, $code = 302)
```

Comment:
Please hold on on this changes until https://github.com/friendica/friendica/pull/7078#discussion_r280979695 is addressed.